attachment_synchronize/models/attachment_synchronize_task.py

```diff
@@ -89,7 +89,7 @@ class AttachmentSynchronizeTask(models.Model):
         "\nFurther operations will be realized on these Attachments Queues depending "
         "on their 'File Type' value.",
     )
-    enabled = fields.Boolean("Enabled", default=True)
+    active = fields.Boolean("Enabled", default=True, old="enabled")
```

Comment:
if task is deactivated will disappear from list, that's why used enabled instead of active. is this a wanted feature?